lib/index.js

```diff
@@ -89,6 +103,7 @@ PostHTML.prototype.process = function (tree, options) {
   tree = options.skipParse ? tree : parser(tree)
 
   tree.options = options
+  tree.processor = this
```

Comment:
> Would the name && version really be needed on the tree?

Your options to reach process ?

```js
+ tree.processor = this
```

This should be in the constructor/processor `ctx.processor.name`
```js
- tree.name = this.name
- tree.version = this.version
```